static/app/actionCreators/tags.tsx

```diff
@@ -150,3 +150,45 @@ export function fetchTagValues({
     query,
   });
 }
+
+export function fetchSpanFieldValues({
+  api,
+  orgSlug,
+  fieldKey,
+  endpointParams,
+  projectIds,
+  search,
+}: {
+  api: Client;
+  fieldKey: string;
+  orgSlug: string;
+  endpointParams?: Query;
+  projectIds?: string[];
+  search?: string;
+}): Promise<TagValue[]> {
+  const url = `/organizations/${orgSlug}/spans/fields/${fieldKey}/values/`;
+
+  const query: Query = {};
+  if (search) {
+    query.query = search;
+  }
```

Comment:
FYI this search doesn't do anything yet but not a big deal for the first pass I think

Reply:
I know but I'm copying the form from tags values and I assume long-terms the apis will have similar shape. No point removing this just to add it in a month or two.